### PR TITLE
additional last resorts for GPL- 1.0 and 2.0

### PR DESCRIPTION
**Pull request by taleinat**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -224,7 +224,9 @@ var lastResorts = [
   ['GNU', 'GPL-3.0-or-later'],
   ['LGPL', 'LGPL-3.0-or-later'],
   ['GPLV1', 'GPL-1.0-only'],
+  ['GPL-1', 'GPL-1.0-only'],
   ['GPLV2', 'GPL-2.0-only'],
+  ['GPL-2', 'GPL-1.0-only'],
```
Comment (kemitchell) [inline index.js]: `GPL-2.0-only`?

Comment (taleinat) [reply index.js]: Bah, yes of course!

**Comment by kemitchell** [comment]
License your stuff under Apache 2?

If so, let me know, and add yourself to contributors in package.json if you like.

It’s late for me. Hopefully publish tomorrow.

**Comment by taleinat** [comment]
Our fork is also Apache-2.0 licensed, so I don't see an issue.

**Comment by taleinat** [comment]
I'll add another test case and myself to the contributors.

**Comment by kemitchell** [comment]
v3.0.1

Thanks again!

I'll add you as a contributor shortly. Feel free to do whatever needs doing, but keep sending PRs for significant changes. I'll do the same.

**Pull request merged**